lib/auth/auth.go

```diff
@@ -3000,7 +3000,12 @@ func (a *Server) isMFARequired(ctx context.Context, checker services.AccessCheck
 		// Check RBAC against all matching nodes and return the first error.
 		// If at least one node requires MFA, we'll catch it.
 		for _, n := range matches {
-			err := checker.CheckAccessToServer(t.Node.Login, n, services.AccessMFAParams{AlwaysRequired: false, Verified: false})
+			err := checker.CheckAccess(
+				n,
+				services.AccessMFAParams{},
+				true,
```

Comment:
Annotate booleans so their meaning is clear?

```suggestion
				true /* matchLabels */,
```

Reply:
or refactor to remove `bool` all together?